### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change

(1) add docs for neuralchat v3-3
(2) modify previous tutorial

### CODE REVIEW

File: docs/reproduce/efficient_LLM_inference_on_cpus.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### FP32 Inference (Baseline)
+
+>**Note**: Please download the corresponding AI model from [huggingface hub](https://huggingface.co/models) before executing following command.
+
+
+``` bash
+cd examples/huggingface/neural_speed
+pip install -r requirements.txt
+numactl -m <node N> -C <cpu list> python run_inference_.py \
```
Comment (WeiweiZhang1) [inline docs/reproduce/efficient_LLM_inference_on_cpus.md:31]: 'run_inference_.py' may be a typo

File: docs/reproduce/neural_chat_v3-3_workflow.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Step-by-Step
+
+This is a step-by-step tutorial to obtain the extreme inference speed and accuracy of [Intel/neural-chat-7b-v3-3](https://huggingface.co/Intel/neural-chat-7b-v3), we recommend using [AutoRound](https://github.com/intel/auto-round.git) for quantization and [Neural Speed](https://github.com/intel/neural-speed.git) for inference.
```
Comment (kevinintel) [inline docs/reproduce/neural_chat_v3-3_workflow.md:3]: add NeuralSpeed version and ITREX version

File: examples/huggingface/neural_speed/README.md [hunk 3]
```diff
@@ -41,33 +41,47 @@ pip install -r requirements.txt
...
->        model_args=f'pretrained="{args.model_name}", dtype=float32',
->        tasks=[f"{args.tasks}"]
->   )
-> ```
+```bash
+# int4 with group-size=32
+python run_accuracy.py \
+    --model_name ./Llama2-fp32 \
+    --tasks "lambada_openai" \
+    --model_format "torch"
```
Comment (WeiweiZhang1) [inline examples/huggingface/neural_speed/README.md:86]: Now,  there's no need for ‘runtime’ format ?

Comment (VincyZhang) [reply examples/huggingface/neural_speed/README.md:86]: To run fp32 accuracy, runtime do not support

File: docs/reproduce/neural_chat_v3-3_workflow.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+
+# Prerequisite​
+We recommend use the latest version of [Intel-Extension-for-Transformers v1.3.1](https://pypi.org/project/intel-extension-for-transformers/1.3.1/) and [NeualSpeed v0.2.dev0](https://pypi.org/project/neural-speed/0.2.dev0/)
+
+```bash
+# install neural-speed
+pip install neural-speed==0.2.dev0
+# install intel-extension-for-transformers
+pip install intel-extension-for-transformers==1.3.1
```
Comment (kevinintel) [inline docs/reproduce/neural_chat_v3-3_workflow.md:13]: autoround?

**Review by a32543254 — APPROVED** [review]
LGTM

File: docs/reproduce/neural_chat_v3-3_workflow.md (hunk 4 above)
Comment (VincyZhang) [reply docs/reproduce/neural_chat_v3-3_workflow.md:13]: installed when quantization:

cd examples/huggingface/pytorch/text-generation/quantization/auto_round
pip install -r requirements.txt
bash run_autoround.sh

Comment (VincyZhang) [reply docs/reproduce/neural_chat_v3-3_workflow.md:13]: https://github.com/intel/intel-extension-for-transformers/blob/main/examples/huggingface/pytorch/text-generation/quantization/auto_round/requirements.txt#L12

**Review by kevinintel — APPROVED** [review]
